core/modules/installer/installer.pages.inc

```diff
@@ -263,11 +263,9 @@ function installer_browser_installation_select_versions_form($form, &$form_state
       if (isset($release['terms']['Release type']) && !empty($release['terms']['Release type'])) {
         $release_title .= check_plain(" (" . implode(', ', $release['terms']['Release type']) . ")");
       }
+      $releases_list[$version] = $release_title;
       if (isset($release['release_link'])) {
-        $releases_list[$version] = l($release_title, $release['release_link'], array('html' => TRUE, 'attributes' => array('target' => '_blank')));
-      }
-      else {
-        $releases_list[$version] = $release_title;
+        $releases_list[$version] .= ' (' . l(t('release notes'), $release['release_link'], array('html' => TRUE, 'attributes' => array('title' => t('Release notes for @project_title', array('@project_title' => $project['title'])), 'target' => '_blank'))) . ')';
```

Comment:
Not entirely sure why we are using `'html' => TRUE` here?

Reply:
I suspect so that the link is rendered?

Reply:
`html` was needed previously because the release title was already escaped, for example if the module name was `It's awesome` the apostrophe would have previously been escaped. Without specifying `html`, the link would have ended up reading `It&#39;s awesome`.

But now we're only linking `release notes`, which hasn't been escaped previously, so we should remove the `html` flag. I've updated the PR.